### DISCUSSION TITLE
zkplugin

### DISCUSSION THREAD

**Discussion by Kd502020** [Cyfrin/foundry-full-course-cu#2038] [Q&A]
please I'm getting this blank screen while trying to set up the zk plugin any one here that can help me
![Screenshot (11)](https://github.com/user-attachments/assets/f378ca88-d897-43c6-9f9d-28378911ffda)

**Comment by EngrPips** [comment]
Hello @Kd502020, Please refresh your browser and retry. That seems weird.

**Reply by Kd502020** [reply]
Did as you said sir but still same issue

**Reply by EngrPips** [reply]
That is weird. Try another browser. I haven't run into this issue before. Can be hard to tell what is wrong.
